components/project-card/project-card-detailed.jsx

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import ReactGA from 'react-ga';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import moment from 'moment';
+import Creators from './meta/creators.jsx';
+import Thumbnail from './meta/thumbnail.jsx';
+import Title from './meta/title.jsx';
+import Description from './meta/description.jsx';
+import WhyInteresting from './meta/why-interesting.jsx';
+import IssuesAndTags from './meta/issues-and-tags.jsx';
+import GetInvolved from './meta/get-involved.jsx';
+import BookmarkControl from '../bookmark-control.jsx';
+import bookmarkManager from '../../js/bookmarks-manager';
+import user from '../../js/app-user.js';
```

Comment:
This many imports suggest that this file is probably doing too many things and we should, in follow-up, refactor it into a few obviously-linked components.